atomicfu-common/src/main/kotlin/kotlinx/atomicfu/Trace.kt

```diff
@@ -0,0 +1,7 @@
+package kotlinx.atomicfu
+
+public expect class Trace(size: Int) {
+    public fun append(text: String)
```

Comment:
@elizarov are we going to opt-in for the logging format by default?
E.g. K/N has something like this in the compiler:
```
val foo = ...
DEBUG {
  // check some expensive foo invariant
}

```

Reply:
Also, it definitely should be `open`

Reply:
It is not so simple. The references to the `Trace` class get erased by the bytecode post-prossor, so you cannot easily really customize it by creating your own subclass. Transformer should have to be smart enough to erase your custom subclasses and their constructors, too. Let's finish (and cleanup) basic implementation first and open a separate issue for that. See #20

Reply:
See #20 for custom formats